src/components/Card/index.stories.tsx

```diff
@@ -275,6 +282,19 @@ export const Large = () => (
   </a>
 )
 
+export const CuratedCard = () => (
+  <ContainerCuratedCard>
+    <Card
+      type="curatedCard"
+      image="https://s3-eu-west-1.amazonaws.com/ticketswap-public/public/201610/80afc84b-2091-4359-9fd7-9126d684182c.jpeg"
+      title="Into the dunes Festival"
+      text="Whisper of the sand in the dunes, sun on your face, feet burning... find the oasis"
+    />
+  </ContainerCuratedCard>
```

Comment:
This won't work without container, then it will look like this:

<img width="1105" alt="CleanShot 2023-05-09 at 15 21 53@2x" src="https://github.com/TicketSwap/solar/assets/14276144/bafea13c-6bd3-4468-b3c9-a2ed9d0a9dc7">